src/main/java/io/cdap/plugin/zendesk/source/batch/ZendeskBatchSourceConfig.java

```diff
@@ -235,22 +236,23 @@ public void validate(FailureCollector collector) {
 
   @VisibleForTesting
   void validateConnection(FailureCollector collector) {
```

Comment:
move this method to connector config class

Reply:
This method is different than the method used for validation in ConnectorConfig. This needs to be placed here only.

Reply:
All 6 properties checked here should be part of connector config and this method should be part of that class.

Reply:
Done